src/06-loops.js

```diff
@@ -61,7 +84,21 @@ function rangeEveryOther(min, max) {}
  *  containsWhileLoop([ "left", "up", "right" ], "down");
  *  //> false
  */
-function containsWhileLoop(array, target) {}
+//didnt pass
+function containsWhileLoop(array, target) {
```

Comment:
So this code should actually pass our tests.  It works and does what you intended.  However, based on the very specific ask of the tests this code actually fails.  One of our tests specifically turns your entire function body into a string.  It then checks if that string has the word ` for ` in it - which our commented out code does.  This is a very strange edge case that I have flagged for our curriculum team but it does also serve as a good reminder that comments in our code do not always exist without consequence.  We should start to remove our pseudo code before we attempt tests moving forward.